parm/post_avblflds.xml

```diff
@@ -4074,7 +4074,7 @@
        <scale>3.0</scale>
     </param>
 
-<!-- 467-469 EDR on FD_HEIGHT_LVL -->
+<!-- 467-469 EDR on FD_HEIGHT_LVL (for regional model)-->
```

Comment:
@YaliMao-NOAA Are you saying ID 467-469 are used for regional models such as RRFS?

Reply:
Yes, these turbulence IDs are used for RAP which outputs turbulence on **flight height** levels, different from GFS outputs turbulence on **pressure levels**
So these IDs are used for a different type of vertical levels

Reply:
@YaliMao-NOAA Thanks for explaining.